core-java-modules/core-java-jvm-2/src/main/java/com/baeldung/loadedclasslisting/ClassLoaderType.java

```diff
@@ -0,0 +1,6 @@
+package com.baeldung.loadedclasslisting;
+
+public enum ClassLoaderType {
+
```

Comment:
remove blank line

Reply:
removed this class ; refactored this to a public enum inside ListLoadedClassesAgent.java